api/mesh/options.proto

```diff
@@ -40,6 +40,9 @@ message KumaResourceOptions {
   // If resource has more than one version, then the flag defines which version
   // is used in the storage. All other versions must be convertible to it.
   bool storage_version = 14;
+
+  // Display Name
```

Comment:
That's not very descriptive. I liked that one
```
	// DisplayName the name of the policy showed as plural to be displayed in the UI and maybe CLI
```

Reply:
IMO a sentence that just restates "this is the name for displaying" is more noisy than helpful. Not every field needs docs.

If we're gonna add the name of the field as the first word, it should be a proper sentence. But I blame godoc's ridiculous convention for that